drivers/claims_reporting/app/models/claims_reporting/calculators/patient_risk_score.rb

```diff
@@ -0,0 +1,159 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module ClaimsReporting::Calculators
+  class PatientRiskScore
+    def initialize(medicaid_ids = [], date_range: Date.current.beginning_of_year..Date.current.end_of_year)
```

Comment:
Does it matter that this goes out into the future?

Reply:
It was done this way to be compatible with what the report did by default. That said, it does mean that the risk scores are going to be weird the beginning of the year on the NCM dashboard -- I'll look at making this a sliding window or something...

Reply:
This turned out to be the wrong risk score, so, leaving this as is, just not using it in the UI.